build_tools/configure_ci.py

```diff
@@ -202,15 +202,15 @@ def should_ci_run_given_modified_paths(paths: Optional[Iterable[str]]) -> bool:
     },
 }
 
-LINUX_BUILD_DEFAULTS = ["gfx94X", "gfx110X"]
-LINUX_TEST_DEFAULTS = ["gfx94X"]
-WINDOWS_BUILD_DEFAULTS = ["gfx110X"]
-WINDOWS_TEST_DEFAULTS = []
+DEFAULT_LABELS = ["gfx94X-linux", "gfx110X-linux", "gfx110X-windows"]
```

Comment:
I would still keep these separate. Labels are _one input_ that feeds into determining which builds to run. So rather than set default "labels", set default "builds" (or "configurations").